pkg/maps/nodemap/node_map_v2.go

```diff
@@ -0,0 +1,125 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package nodemap
+
+import (
+	"fmt"
+	"net"
+	"unsafe"
+
+	"golang.org/x/sys/unix"
+
+	"github.com/cilium/cilium/pkg/ebpf"
+)
+
+// compile time check of MapV2 interface
+var _ MapV2 = (*nodeMapV2)(nil)
+
+const (
+	MapNameV2 = "cilium_node_map_v2"
```

Comment:
I think we also need to ensure that we dump both maps in the bugtool. Just run `git grep -W v2 bugtool/` and you'll see what I mean :sweat_smile:

Reply:
Oh, crap, you're right! Let's handle that in an ad-hoc PR then.